chains/substrate/listener/deposit-handler.go

```diff
@@ -6,14 +6,14 @@ package listener
 import (
 	"errors"
 
-	"github.com/rs/zerolog/log"
-
-	"github.com/ChainSafe/chainbridge-core/relayer/message"
-	core_types "github.com/ChainSafe/chainbridge-core/types"
+	"github.com/ChainSafe/sygma-relayer/chains"
+	"github.com/ChainSafe/sygma-relayer/chains/substrate"
 	"github.com/centrifuge/go-substrate-rpc-client/v4/types"
+	"github.com/rs/zerolog/log"
+	"github.com/sygmaprotocol/sygma-core/relayer/message"
 )
 
-type DepositHandlers map[message.TransferType]DepositHandlerFunc
+type DepositHandlers map[message.MessageType]DepositHandlerFunc
```

Comment:
Why was this changed to `MessageType`. It seems that `TransferType` makes more sense in this context.

Reply:
The deposit handler then creates the message

Reply:
`TransferType` existed in `chainbridge-core` so I've used `MessageType` from `sygma-core` because both are aliases for the built-in type `string`. But I agree, it is better naming to use `TransferType`, so I'll make a custom one in the repo?

Reply:
yep